dissect/target/plugins/filesystem/walkfs.py

```diff
@@ -80,9 +113,19 @@ def generate_record(target: Target, entry: FilesystemEntry) -> FilesystemRecord:
     stat = entry.lstat()
 
     if isinstance(entry, LayerFilesystemEntry):
-        fs_types = [sub_entry.fs.__type__ for sub_entry in entry.entries]
+        fs_types = []
+        volume_uuids = []
+        disk_serials = []
+
+        for sub_entry in entry.entries:
+            fs_types.append(sub_entry.fs.__type__)
+            volume_uuids.append(get_volume_uuid(sub_entry.fs))
```

Comment:
```suggestion
            volume_uuids.append(sub_entry.fs.uuid)
```
Can't this whole `get_volume_uuid` function be replaced like this? Because the uuid attribute is now defined in `Filesystem`